### PR TITLE
Update tiflash-configuration.md

### PR DESCRIPTION
**Pull request by jinshayumi**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions))

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -193,6 +193,9 @@ delta_index_cache_size = 0
     ## 在 v6.2.0 以及后续版本，强烈建议保留默认值 `false`，不要将其修改为 `true`。具体请参考已知问题 [#5576](https://github.com/pingcap/tiflash/issues/5576)。
     # dt_enable_logical_split = false
 
+    ## `max_threads`: 默认值为系统的 CPU 核数，TiFlash 执行最大线程数，
+    ## 此参数只会在系统变量 `tidb_max_tiflash_threads` 设置为 -1 的时候生效
+
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:198]:
```suggestion
    # max_threads = 32
```

**Review by hfxsd — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## 在 v6.2.0 以及后续版本，强烈建议保留默认值 `false`，不要将其修改为 `true`。具体请参考已知问题 [#5576](https://github.com/pingcap/tiflash/issues/5576)。
     # dt_enable_logical_split = false
 
+    ## `max_threads`: 执行一个 MMP Task 的线程并发度，默认值为 0，当值为 0 时，该值被设置为 CPU 核数。
+    ## 此参数只会在系统变量 `tidb_max_tiflash_threads` 设置为 -1 的时候生效。
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:197]:
```suggestion
    ## 该参数只有在系统变量 `tidb_max_tiflash_threads` 设置为 -1 时才会生效。
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## 在 v6.2.0 以及后续版本，强烈建议保留默认值 `false`，不要将其修改为 `true`。具体请参考已知问题 [#5576](https://github.com/pingcap/tiflash/issues/5576)。
     # dt_enable_logical_split = false
 
+    ## `max_threads`: 执行一个 MMP Task 的线程并发度，默认值为 0，当值为 0 时，该值被设置为 CPU 核数。
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md]:
```suggestion
    ## `max_threads` 指的是执行一个 MMP Task 的线程并发度，默认值为 0。当值为 0 时，TiFlash 执行 MMP Task 的线程并发度为 CPU 核数。
```

Comment (hfxsd) [inline tiflash/tiflash-configuration.md:196]:
```suggestion
    ## `max_threads`: 执行一个 MMP Task 的内部线程并发度，默认值为 0，当值为 0 时，该值被设置为 CPU 核数。
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -193,6 +193,10 @@ delta_index_cache_size = 0
     ## 在 v6.2.0 以及后续版本，强烈建议保留默认值 `false`，不要将其修改为 `true`。具体请参考已知问题 [#5576](https://github.com/pingcap/tiflash/issues/5576)。
     # dt_enable_logical_split = false
 
+    ## `max_threads`: 执行一个 MMP Task 的内部线程并发度，默认值为 0，当值为 0 时，该值被设置为 CPU 核数。
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:196]:
```suggestion
    ## `max_threads` 指的是执行一个 MMP Task 的内部线程并发度，默认值为 0。当值为 0 时，TiFlash 执行 MMP Task 的线程并发度为 CPU 核数。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**